examples/config_templates/table_builder/mcap_ros2.yaml

```diff
@@ -0,0 +1,51 @@
+#@yaml/text-templated-strings
+---
+_target_: rbyte.io.table.TableBuilder
+reader:
+  _target_: rbyte.io.table.mcap.McapRos2TableReader
+  _recursive_: false
+  _convert_: all
+  fields:
+    /nissan/vehicle_speed:
+      data:
+
+    /nissan/gps/duro/current_pose:
+      pose:
+        position:
+          x:
+          "y":
+
+        orientation:
+          x:
+          "y":
+
+merger:
+  _target_: rbyte.io.table.TableMerger
+  separator: /
+  merge:
+    /nissan/vehicle_speed:
+      log_time:
+        method: ref
+
+    /nissan/gps/duro/current_pose:
+      log_time:
+        method: ref
```

Comment:
This is for which dataset ?

Reply:
https://jkk-research.github.io/dataset/jkk_dataset_02/

added link to yaml